extensions/amp-date-picker/0.1/date-range-picker.js

```diff
@@ -169,21 +169,6 @@ function createDateRangePickerBase() {
      * @param {?string} focusedInput
      */
     onFocusChange(focusedInput) {
-      // TODO(cvializ): there is a bug in the react-dates library that
```

Comment:
This bug only happened when using `preact-compat` with `react-dates`, so this workaround code isn't needed any more.